dev-db/erd/erd-0.2.1.0.ebuild

```diff
@@ -29,3 +29,10 @@ DEPEND="${RDEPEND}
 		>=dev-haskell/tasty-hunit-0.9 )
 "
 PATCHES=("${FILESDIR}/${PN}-0.2.1.0-ghc-9.patch")
+
```

Comment:
`cabal_chdeps` was deprecated in 0bcd1a1e80155c98ec3a2304e4b6cf9e763b9d64. Please use `CABAL_CHDEPS` instead. `haskell-cabal_src_prepare` now does quite a bit so overriding it like this isn't a good idea anymore.